packages/widget/src/components/transfer/fungible/transfer-detail/transfer-detail.ts

```diff
@@ -49,19 +56,46 @@ export class FungibleTransferDetail extends BaseComponent {
     }
   }
 
-  getFee(): string {
+  getGasFeeParams(): {
+    decimals?: number;
+    symbol: string;
+  } {
+    let decimals = undefined;
```

Comment:
nit and question: if we use default values for the parameters, would this two declarations make sense? 🤔

Reply:
what do you mean by default values?

Reply:
resolved, nvm this comment